Sitefinity-VSIX/vNextPackage.cs

```diff
@@ -11,11 +11,11 @@
 
 namespace Sitefinity_VSIX
 {
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(AllowsBackgroundLoading = true, UseManagedResourcesOnly = true)]
```

Comment:
What does this flag do?

Reply:
Marks the package as safe to be loaded on a background thread. 
If it is not set to true Visual Studio 2019 shows warning that the extension is using deprecated APIs. 
![vs2019](https://user-images.githubusercontent.com/12613338/60958300-cfa3d100-a30e-11e9-9c40-6ba07cce6ce8.PNG)